library/src/main/java/com/pokegoapi/api/player/PlayerProfile.java

```diff
@@ -305,4 +293,34 @@ public Stats getStats()
 		}
 		return stats;
 	}
+
+	/**
+	 * Gets tutorial states
+	 *
+	 * @return TutorialState object
+	 * @throws LoginFailedException when the auth is invalid
+	 * @throws RemoteServerException when the server is down/having issues
+	 */
+	public TutorialState getTutorialState()
+			throws LoginFailedException, RemoteServerException {
+		return tutorialState;
+	}
+
+	public void enableAccount() throws LoginFailedException, RemoteServerException {
+		MarkTutorialCompleteMessage.Builder tutorialBuilder = MarkTutorialCompleteMessage.newBuilder();
+		tutorialBuilder.addTutorialsCompleted(TutorialStateOuterClass.TutorialState.LEGAL_SCREEN)
+				.setSendMarketingEmails(false)
+				.setSendPushNotifications(false);
+
+		ServerRequest serverRequest = new ServerRequest(RequestType.MARK_TUTORIAL_COMPLETE, tutorialBuilder.build());
+		api.getRequestHandler().sendServerRequests(serverRequest);
+
+		MarkTutorialCompleteResponse response;
+		try {
+			response = MarkTutorialCompleteResponse.parseFrom(serverRequest.getData());
+			Log.d("tutooo", response.toString());
```

Comment:
No, these are my tests! I added it to check if the response was successfull and if you run it and filter for tutooo on logcat you should see the result. Once i got some score by you i'll cleanup the code. Note that im building it with terminal and without an IDE so if you see any other incorrect import, just tell me